api/src/queries/project/project-delete-queries.ts

```diff
@@ -263,6 +263,27 @@ export const deleteProjectRegionSQL = (projectId: number | undefined): SQLStatem
   return sqlStatement;
 };
 
+/**
+ * SQL query to delete range associated to a project ID.
+ *
+ * @param {number} projectId
+ * @returns {SQLStatement} sql query object
+ */
+export const deleteProjectRangeSQL = (projectId: number | undefined): SQLStatement | null => {
```

Comment:
Do we actually expect projectId to be undefined sometimes?
Or can we remove the `undefined` option, and just presume the caller gives us a number, and then the if check below can be removed.